test/features/parsoid/transform.js

```diff
@@ -76,12 +76,37 @@ function findSpecs() {
 }
 
 function x2y(spec) {
+
+    function makeReq(filename) {
+        var req = {
+          uri: server.config.baseURL + '/transform/' + spec.from.format +
+               '/to/' + spec.to.format,
+        };
+        if (/\.json$/.test(filename)) {
+            req.headers = { 'content-type': 'application/json' };
+            req.body = readFile(spec.from.src);
+        } else if (/\.html$/.test(filename)) {
+            req.headers = {
+              'content-type': 'multipart/form-data; ' +
```

Comment:
It might actually be enough to set the content-type to multipart/form-data and provide a structured body here. The request library should support multipart/form-data encoding (and if it doesn't, then we should fix that in preq).